src/components/structural/header/Header.js

```diff
@@ -292,35 +272,31 @@ class Header extends Component {
     * @summary - This sets the components current state to the input from the scene name form
     */
     handleNameChange = (event) => {
-        this.setState({ sceneName: event.target.value });
+        this.props.sceneActions.nameScene(event.target.value);
     }
 
     /**
     * @summary - This sets the components current state to the input from the scene description form
     */
     handleDescChange = (event) => {
-        this.setState({ sceneDesc: event.target.value });
+        this.props.sceneActions.setDesc(event.target.value);
     }
 
     /**
     * @summary - This function produces the form for inputting the scene's name and description
     */
     sceneName = () => {
-        let text = "";
-        if (this.state.sceneName === null) {
-            text = this.props.scene.name;
-        } else {
-            text = this.state.sceneName;
-        }
+        let nameText = this.props.scene.name;
```

Comment:
Can we name these sceneName and sceneDesc? It's not immediately clear on what these variables when you first read them.